### PR TITLE
Changed subscription status check to be stored on session storage

### PR DESCRIPTION
**Pull request by rodrigopavezi**
- Changed it to use Store Authorization API
- Fixed tests

@donnapep @stulees please review.

Image Widget presentation here:

http://rva.risevision.com/#/PRESENTATION_MANAGE/id=2988d6fe-6647-46c5-9a7b-9ab3da8e8dcf?cid=30007b45-3df0-4c7b-9f7f-7d8ce6443013

You can notice that it only makes the calls to store once and sometimes only one per company id even though there are 2 widgets with a file from the same company.

cheers

### CODE REVIEW

File: rise-storage.html [hunk 1]
```diff
@@ -334,24 +349,96 @@
...
     },
 
+    _makeStorageRequest: function() {
+      this._computeStorageUrl();
+      this._hasAttemptedSecondStorageRequest = false;
+      this._loadStorage();
+    },
+
+    _fireStorageSubscriptionError: function() {
+      console.log("Storage Subscription Expired!");
```
Comment (stulees) [inline rise-storage.html]: @rodrigopavezi Do we need this console log?

File: rise-storage.html [hunk 2]
```diff
@@ -334,24 +349,96 @@
...
+          }
+        }
+      } else {
+        if (this._isLoading) {
+          this._makeStorageSusbscriptionRequest();
+        } else {
+          if (this._isSubscribed) {
+            this._makeStorageRequest();
+          } else {
+            this._fireStorageSubscriptionError();
```
Comment (stulees) [inline rise-storage.html]: @rodrigopavezi Do you think we need to fire this again?

Comment (rodrigopavezi) [inline rise-storage.html]: @stulees good point, we don't really need to fire it on a refresh. I will remove it. cheers

File: rise-storage.html (hunk 1 above)
Comment (rodrigopavezi) [inline rise-storage.html]: @stulees It can be removed. cheers

**Comment by rodrigopavezi** [comment]
@stulees changes made. please review. cheers

**Comment by stulees** [comment]
@rodrigopavezi Looks good 👍

**Pull request merged**